index.html

```diff
@@ -15,276 +15,291 @@
-                        <article class="text3">javascript
-                    </li>
+                <ul class="nav-links">
+                    <li class="port1"><a href="#portfolio">Portfolio </a></li>
+                    <li class="ab1"><a href="#about">About </a></li>
+                    <li class="cont1"><a href="#contact">Contact</a></li>
                 </ul>
-                <div class="action">
-                    <button class="btn">See Project</button>
-                </div>
-            </div>
-
-        </section>
-        <section class="section2">
-            <div class="right-block00">
-                <img class="img-section2" src="images/Snapshoot Portfolio1.png" alt="project image">
-                <img class="img-sec2" src="images/Snapshoot Portfolio section2.png" alt="project image">
-            </div>
-            <div class="left-block00">
-                <div class="experience1">
-                    <h1 class="section-title1">Multi-Post
-                        Stories</h1>
-                    <ul class="list">
-                        <li class="list1">CANOPY</li>
-                        <li class="list2">Back End Dev</li>
-                        <li class="list3">2015</li>
+                <div class="menu-items">
+                    <ul class="nav-links1" id="menu">
+                        <li><a class="h-menu" href="#portfolio">Portfolio </a></li>
+                        <li><a class="h-menu" href="#about">About </a></li>
+                        <li><a class="h-menu" href="#contact">Contact</a></li>
                     </ul>
                 </div>
-                <p class="section-description0">A daily selection of privately personalized reads; no
-                    accounts or sign-ups
-                    required.</p>
+            </nav>
+        </header>
+    </section>
+    <section class="introduction">
+        <h1 class="title"> I'm Lawrence Glad to See You!</h1>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded,don’t
+            hestiate to contact me.</p>
+        <h2 class="social order">LET'S CONNECT</h2>
+        <ul class="social-links">
+            <li><a href="https://twitter.com/lawrenc98789206"><i class="fa-brands fa-twitter li1"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/lawrence-kioko-972035240/"><i
+                        class="fa-brands fa-linkedin li2"></i></a></li>
+            <li><a href="https://medium.com/@lorenkioko"><i class="fa-brands fa-m li3"></i></a></li>
+            <li><a href="https://github.com/Kidd254"><i class="fa-brands fa-github li4"></i></a></li>
+            <li><a href="https://angel.co/jobs?ref=onboarding"><i class="fa-brands fa-angellist li5"></i></a></li>
+        </ul>
+    </section>
+
+    <section class="section1">
```

Comment:
[OPTIONAL]The main goal of this project is to remove the projects section from the HTML file and generate it dynamically with JavaScript. In your HTML file, there should be only an empty container to target from your JavaScript selector, and the rest need to be generated dynamically as you did in the modal. Check this [repo](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) to understand requirements better 🦾